integration-tests/test_registration.py

```diff
@@ -1,22 +1,91 @@
-import os
-
```

Comment:
better avoid this kind of style change in the same commit -- simply add the other stdlib imports here, rather than reodering things